### PR TITLE
Image input

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# Image and input Component
## Responsiveness:
-Mobile[❎]
-Medium Screen[✔]
-Large Screen[✔]

### CODE REVIEW

**Comment by ashifkhn** [comment]
looks good

**Comment by kirti2602** [comment]
> # Card Component
> ## Sandbox link :[ click here](https://codesandbox.io/s/github/dusariudaykumar/ReadyMade-UI/tree/Image-Input)
> ## Responsiveness:
> -Mobile[❎] -Medium Screen[✔] -Large Screen[✔]

branch says `image-input`, and PR description says Card component?

**Comment by dusariudaykumar** [comment]
I just copied my previous template . I forgot to update it.

**Comment by dusariudaykumar** [comment]
> > # Card Component
> > ## Sandbox link :[ click here](https://codesandbox.io/s/github/dusariudaykumar/ReadyMade-UI/tree/Image-Input)
> > ## Responsiveness:
> > -Mobile[❎] -Medium Screen[✔] -Large Screen[✔]
> 
> branch says `image-input`, and PR description says Card component?



> > # Card Component
> > ## Sandbox link :[ click here](https://codesandbox.io/s/github/dusariudaykumar/ReadyMade-UI/tree/Image-Input)
> > ## Responsiveness:
> > -Mobile[❎] -Medium Screen[✔] -Large Screen[✔]
> 
> branch says `image-input`, and PR description says Card component?

I just copied my previous PR template . I forgot to update it.

File: Components/Images/images.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@import url("/CSS/main.css");
+/* * {
+  outline: 1px solid red;
+} */
+.image-wrapper {
+  display: flex;
```
Comment (kirti2602) [inline Components/Images/images.css:6]: You can try making `display : flex` a utility class and reuse it whenever possible

File: Components/Images/images.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  flex-wrap: wrap;
```
Comment (kirti2602) [inline Components/Images/images.css:7]: Can make utility out of `flex-wrap: wrap` as well

File: Components/Images/images.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  border: 2px solid var(--light-grey);
+  align-items: baseline;
+  border-radius: 0.3rem;
+  width: 100%;
+}
+.responsive-img {
+  width: 100%;
+  height: auto;
+}
+.r-image {
```
Comment (kirti2602) [inline Components/Images/images.css]: Classname `r-image` isn't very descriptive. So maybe try using class names that are self-descriptive like `responsive-img`, `round-image` etc depending of what your class is doing

**Review by kirti2602 — COMMENTED** [review]
Overall looks fine.

File: Components/Images/images.css (hunk 2 above)
Comment (dusariudaykumar) [reply Components/Images/images.css:7]: thanks for your suggestion, i will change it

**Pull request merged**